### ISSUE TITLE
File upload hangs if target is not the root directory

### ISSUE THREAD

**Issue by JPL1** [badcodeltd/safecms#20]
Uploading an image to the root of a domain or subdomain works fine, but if you prepend a folder  "/image/" or "image/" to the File URL field it hangs.
![stuck-uploading](https://user-images.githubusercontent.com/12783198/36696726-ec8fee80-1b3c-11e8-8501-1196a5531c47.png)
 Using Debian 9

EDIT: it's probably not possible to create a folder on SAFE in that way, but it would be good if it threw an error if it sees a "/"

**Comment by BadCodeShane** [comment]
Confirmed, thanks for this, good spot.

On a side note, nice to see the network primer making it's way on to the network itself! 👍

**Comment by JPL1** [comment]
> On a side note, nice to see the network primer making it's way on to the network itself! 

Thanks! - that's my challenge - see if I can come up with a nice looking template for it. BTW is there a way of triggering a browser to open up the internet (http:) links  do you know?

**Comment by BadCodeShane** [comment]
I believe simply adding an anchor tag with the href you want should do it, do they not trigger Chrome/Firefox to open? Normally, those programs add a `system-uri` intercepting any calls to `http(s?)`. If they aren't working, that's probably an issue for these chaps: https://github.com/maidsafe/safe_browser

**Comment by JPL1** [comment]
> I believe simply adding an anchor tag with the href you want should do it, do they not trigger Chrome/Firefox to open? 

Yes, working now. Strange  - I must have made a mistake somewhere.
